jupyter-widgets-htmlmanager/src/htmlmanager.ts

```diff
@@ -45,7 +49,7 @@ class HTMLManager extends widgets.ManagerBase<HTMLElement> {
      */
     protected loadClass(className: string, moduleName: string, moduleVersion: string) {
         return new Promise(function(resolve, reject) {
-            if (moduleName === '@jupyter-widgets/controls') {
+            if (coreWidgetModules.indexOf(moduleName) >= 0) {
```

Comment:
We should provide the widgets in separate cases, as in https://github.com/jupyter-widgets/ipywidgets/blob/master/widgetsnbextension/src/manager.js#L97 (and import the base widgets as a separate namespace).